dask_expr/_resample.py

```diff
@@ -125,14 +193,66 @@ def __init__(self, obj, rule, **kwargs):
     def _single_agg(
         self,
         expr_cls,
+        how_args=ResampleReduction._defaults["how_args"],
```

Comment:
I'd rather not access the _defaults variable here, you can either be explicit or do a **kwargs (I'd prefer being explicit)

Reply:
Let it be explicit then. :+1:  https://github.com/dask-contrib/dask-expr/pull/399/commits/0f6fba3be04d459088e55c837dc927a3215a8cdc